{
  "uid" : "c8f4cef80092e259",
  "name" : "test_returns_true",
  "fullName" : "tests.brownie.token.tests.test_transfer#test_returns_true",
  "historyId" : "f88955ea8db917ca8c4bcbf081df7eef",
  "time" : {
    "start" : 1639070391799,
    "stop" : 1639070403078,
    "duration" : 11279
  },
  "status" : "broken",
  "statusMessage" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
  "statusTrace" : "accounts = <brownie.network.account.Accounts object at 0x7fb72ec92100>\ntoken = <Token Contract '0x9FD5FCC8A4F61F24c0613456Dd2e460F1Ab75278'>\n\n    def test_returns_true(accounts, token):\n        amount = token.balanceOf(accounts[0])\n        tx = token.transfer(accounts[1], amount, {'from': accounts[0]})\n    \n>       assert tx.return_value is True\n\ntests/brownie/token/tests/test_transfer.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/transaction.py:53: in wrapper\n    raise exc\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/transaction.py:49: in wrapper\n    return fn(self)\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/transaction.py:254: in return_value\n    self._get_trace()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Transaction '\u001B[0;m0x557023d2dab200327fc8bfb5e9a210215a16e5b56ab526fb33ee5e4da645fc8e\u001B[0;m'>\n\n    def _get_trace(self) -> None:\n        \"\"\"Retrieves the stack trace via debug_traceTransaction and finds the\n        return value, revert message and event logs in the trace.\n        \"\"\"\n    \n        # check if trace has already been retrieved, or the tx warrants it\n        if self._raw_trace is not None:\n            return\n        self._raw_trace = []\n        if self.input == \"0x\" and self.gas_used == 21000:\n            self._modified_state = False\n            self._trace = []\n            return\n    \n        if not web3.supports_traces:\n            raise RPCRequestError(\"Node client does not support `debug_traceTransaction`\")\n        try:\n            trace = web3.provider.make_request(  # type: ignore\n                \"debug_traceTransaction\", (self.txid, {\"disableStorage\": CONFIG.mode != \"console\"})\n            )\n        except (requests.exceptions.Timeout, requests.exceptions.ConnectionError) as e:\n            msg = f\"Encountered a {type(e).__name__} while requesting \"\n            msg += \"`debug_traceTransaction`. The local RPC client has likely crashed.\"\n            if CONFIG.argv[\"coverage\"]:\n                msg += \" If the error persists, add the `skip_coverage` marker to this test.\"\n            raise RPCRequestError(msg) from None\n    \n        if \"error\" in trace:\n            self._modified_state = None\n            self._trace_exc = RPCRequestError(trace[\"error\"][\"message\"])\n>           raise self._trace_exc\nE           brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'\n\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/transaction.py:668: RPCRequestError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "token",
    "time" : {
      "start" : 1639070325053,
      "stop" : 1639070353849,
      "duration" : 28796
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
    "statusTrace" : "accounts = <brownie.network.account.Accounts object at 0x7fb72ec92100>\ntoken = <Token Contract '0x9FD5FCC8A4F61F24c0613456Dd2e460F1Ab75278'>\n\n    def test_returns_true(accounts, token):\n        amount = token.balanceOf(accounts[0])\n        tx = token.transfer(accounts[1], amount, {'from': accounts[0]})\n    \n>       assert tx.return_value is True\n\ntests/brownie/token/tests/test_transfer.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/transaction.py:53: in wrapper\n    raise exc\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/transaction.py:49: in wrapper\n    return fn(self)\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/transaction.py:254: in return_value\n    self._get_trace()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Transaction '\u001B[0;m0x557023d2dab200327fc8bfb5e9a210215a16e5b56ab526fb33ee5e4da645fc8e\u001B[0;m'>\n\n    def _get_trace(self) -> None:\n        \"\"\"Retrieves the stack trace via debug_traceTransaction and finds the\n        return value, revert message and event logs in the trace.\n        \"\"\"\n    \n        # check if trace has already been retrieved, or the tx warrants it\n        if self._raw_trace is not None:\n            return\n        self._raw_trace = []\n        if self.input == \"0x\" and self.gas_used == 21000:\n            self._modified_state = False\n            self._trace = []\n            return\n    \n        if not web3.supports_traces:\n            raise RPCRequestError(\"Node client does not support `debug_traceTransaction`\")\n        try:\n            trace = web3.provider.make_request(  # type: ignore\n                \"debug_traceTransaction\", (self.txid, {\"disableStorage\": CONFIG.mode != \"console\"})\n            )\n        except (requests.exceptions.Timeout, requests.exceptions.ConnectionError) as e:\n            msg = f\"Encountered a {type(e).__name__} while requesting \"\n            msg += \"`debug_traceTransaction`. The local RPC client has likely crashed.\"\n            if CONFIG.argv[\"coverage\"]:\n                msg += \" If the error persists, add the `skip_coverage` marker to this test.\"\n            raise RPCRequestError(msg) from None\n    \n        if \"error\" in trace:\n            self._modified_state = None\n            self._trace_exc = RPCRequestError(trace[\"error\"][\"message\"])\n>           raise self._trace_exc\nE           brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'\n\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/transaction.py:668: RPCRequestError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "328d57c55087da93",
      "name" : "stdout",
      "source" : "328d57c55087da93.txt",
      "type" : "text/plain",
      "size" : 312
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Brownie"
  }, {
    "name" : "parentSuite",
    "value" : "tests.brownie.token.tests"
  }, {
    "name" : "suite",
    "value" : "test_transfer"
  }, {
    "name" : "host",
    "value" : "fv-az123-316"
  }, {
    "name" : "thread",
    "value" : "4719-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.brownie.token.tests.test_transfer"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Not supported",
      "messageRegex" : ".*'EthereumModel' object has no attribute .*",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 56,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 56
      },
      "items" : [ {
        "uid" : "6e6df2a4c45e4205",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/190//#testresult/6e6df2a4c45e4205",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1639074284554,
          "stop" : 1639074284554,
          "duration" : 0
        }
      }, {
        "uid" : "cb0182385bc09954",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/187//#testresult/cb0182385bc09954",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1639053227388,
          "stop" : 1639053231488,
          "duration" : 4100
        }
      }, {
        "uid" : "8feb0e5d9e302d7e",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/188//#testresult/8feb0e5d9e302d7e",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639056744836,
          "stop" : 1639056776272,
          "duration" : 31436
        }
      }, {
        "uid" : "12510d44d37242d9",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/186//#testresult/12510d44d37242d9",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1639045821541,
          "stop" : 1639045823669,
          "duration" : 2128
        }
      }, {
        "uid" : "96f80a1beb397846",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/184//#testresult/96f80a1beb397846",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ReadTimeout: HTTPSConnectionPool(host='proxy.teststand2.neontest.xyz', port=443): Read timed out. (read timeout=30)",
        "time" : {
          "start" : 1639019925572,
          "stop" : 1639019925572,
          "duration" : 0
        }
      }, {
        "uid" : "74c655c0b98543eb",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/183//#testresult/74c655c0b98543eb",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639010354597,
          "stop" : 1639010386015,
          "duration" : 31418
        }
      }, {
        "uid" : "7b8a7d63d19087bb",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/182//#testresult/7b8a7d63d19087bb",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638990652450,
          "stop" : 1638990654902,
          "duration" : 2452
        }
      }, {
        "uid" : "14767e58d5ecec31",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/177//#testresult/14767e58d5ecec31",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638969790999,
          "stop" : 1638969798877,
          "duration" : 7878
        }
      }, {
        "uid" : "4100ace2e4a6dd95",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/181//#testresult/4100ace2e4a6dd95",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1638981014241,
          "stop" : 1638981014241,
          "duration" : 0
        }
      }, {
        "uid" : "a1647abcdb4b2eaf",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/180//#testresult/a1647abcdb4b2eaf",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638975772663,
          "stop" : 1638975774280,
          "duration" : 1617
        }
      }, {
        "uid" : "9b59fc9aef4e8e55",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/179//#testresult/9b59fc9aef4e8e55",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638974631287,
          "stop" : 1638974633736,
          "duration" : 2449
        }
      }, {
        "uid" : "15f8b85a53dd1314",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/178//#testresult/15f8b85a53dd1314",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638973510246,
          "stop" : 1638973512904,
          "duration" : 2658
        }
      }, {
        "uid" : "a14c532d9cb659af",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/176//#testresult/a14c532d9cb659af",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638951164715,
          "stop" : 1638951172422,
          "duration" : 7707
        }
      }, {
        "uid" : "24def6181b8e11e6",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/174//#testresult/24def6181b8e11e6",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.teststand2.neontest.xyz', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fadbd170670>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638932974513,
          "stop" : 1638932974513,
          "duration" : 0
        }
      }, {
        "uid" : "6cd50d371b1b55b6",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/173//#testresult/6cd50d371b1b55b6",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.teststand2.neontest.xyz', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb865a4d640>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638923383261,
          "stop" : 1638923383261,
          "duration" : 0
        }
      }, {
        "uid" : "1fe0278637904fb4",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/172//#testresult/1fe0278637904fb4",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638913187778,
          "stop" : 1638913193953,
          "duration" : 6175
        }
      }, {
        "uid" : "245413d29a55df4b",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/171//#testresult/245413d29a55df4b",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.testnet.neonlabs.org', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2408972670>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638912576538,
          "stop" : 1638912576538,
          "duration" : 0
        }
      }, {
        "uid" : "e9cb8481a3f3925f",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/170//#testresult/e9cb8481a3f3925f",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638908569608,
          "stop" : 1638908575353,
          "duration" : 5745
        }
      }, {
        "uid" : "15e3a61513e76b2f",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/169//#testresult/15e3a61513e76b2f",
        "status" : "broken",
        "statusDetails" : "brownie.exceptions.RPCRequestError: 'EthereumModel' object has no attribute 'debug_traceTransaction'",
        "time" : {
          "start" : 1638892660522,
          "stop" : 1638892662073,
          "duration" : 1551
        }
      }, {
        "uid" : "70beb2e3012d7e7e",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/168//#testresult/70beb2e3012d7e7e",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1638891850552,
          "stop" : 1638891850552,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "c8f4cef80092e259.json",
  "parameterValues" : [ ]
}